PhysicsTools/HeppyCore/python/framework/config.py

```diff
@@ -97,6 +104,12 @@ def build_name(self):
         name = '_'.join([class_name, self.instance_label])
         return name 
 
+    def clone(self, **kwargs):
```

Comment:
Ok, I see, thanks for your answer. 

Yes, a docstring would help. It should provide an example, and state that this function is doing a simple copy, and then sets the attributes provided as kwargs. 

Btw wouldn't it make sense to check that the attribute exists and raise an exception if not? That would catch typos like providing pTmin=40 instead of ptmin=40 (pt being the attribute to be modified).

About the import, I don't think we'll need to create a copy function in this module, so I'd prefer to put it at the top of the module. The main reason why I'm insisting is that I've seen inexperienced python developers systematically put imports within deeper contexts simply because they think it's the best way to proceed. The code then becomes more difficult to maintain when comes the time of rationalizing import statements, and the execution is in principle slower. So I think it's wise not to show them that such imports are possible if we can avoid it :-)